packages/cli/src/create-app/runNpmInstall.cjs

```diff
@@ -0,0 +1,51 @@
+const { spawnSync } = require( 'child_process' );
+const path = require( 'path' );
+
+/**
+ * Runs npm install silently in the project directory.
+ *
+ * @param {string} projectDirPath - Path to the project directory.
+ * @return {Promise<void>}
+ */
+const runNpmInstall = async ( projectDirPath ) => {
+	console.log(
+		'Installing NPM dependencies... This may take a few minutes.'
```

Comment:
- Use [ora](https://github.com/sindresorhus/ora) to add a spinner to the message.
- Change the message to say "Installing NPM dependencies".
- Show the message after spawning is successful.